state-chain/pallets/cf-ingress-egress/src/tests.rs

```diff
@@ -1671,3 +1671,140 @@ fn safe_mode_prevents_deposit_channel_creation() {
 		);
 	});
 }
+
+#[test]
+fn do_not_batch_more_transfers_than_the_limit_allows() {
+	new_test_ext().execute_with(|| {
+		MockFetchesTransfersLimitProvider::enable_limits(true);
+
+		const EXCESS_TRANSFERS: usize = 1;
+		let transfer_limits = MockFetchesTransfersLimitProvider::maybe_transfers_limit().unwrap();
+
+		for _ in 1..=transfer_limits + EXCESS_TRANSFERS {
+			assert_ok!(IngressEgress::schedule_egress(ETH_ETH, 1_000, ALICE_ETH_ADDRESS, None));
+		}
+
+		let sheduled_egresses = ScheduledEgressFetchOrTransfer::<Test, ()>::get();
```

Comment:
```suggestion
		let scheduled_egresses = ScheduledEgressFetchOrTransfer::<Test, ()>::get();
```
think @dandanlen made a typo in his comment...

Reply:
🙈 yep sorry